src/api-reference/financial-integration/v4.financial-integration.markdown

```diff
@@ -141,23 +148,78 @@ Content-Length: 491
 
 ```json
 {
-  "fIDocuments": [
-    {
-      "id": "1234507145154c6b9a1e562cc6b12345",
-      "docType": "expense",
-      "entityId": "YOUR ENTITY CODE",
-      "companyUuid": "YOUR_COMPANY_UUID",
-      "erpSystemId": null,
-      "document": "{THE_FINANCIAL_DOCUMENT}",
-      "docStatus": "READY"
-    }
-  ],
-  "offset": "231234",
-  "nextPage": "/fi/v4/companies/transactiontypes/expense/transactions?offset=231234&limit=100",
-  "numberReturned": 1
+  "links" : [ {
+    "rel" : "first",
+    "href" : "https://fiserviceurlhere/fi/v4/companies/transactiontypes/expense/transactions?limit=3&page=0&size=3",
+    "hreflang" : null,
+    "media" : null,
+    "title" : null,
+    "type" : null,
+    "deprecation" : null
+  }, {
+    "rel" : "self",
+    "href" : "https://fiserviceurlhere/fi/v4/companies/transactiontypes/expense/transactions?limit=3&page=0&size=3",
+    "hreflang" : null,
+    "media" : null,
+    "title" : null,
+    "type" : null,
+    "deprecation" : null
+  }, {
+    "rel" : "next",
+    "href" : "https://fiserviceurlhere/fi/v4/companies/transactiontypes/expense/transactions?limit=3&page=1&size=3",
+    "hreflang" : null,
+    "media" : null,
+    "title" : null,
+    "type" : null,
+    "deprecation" : null
+  }, {
+    "rel" : "last",
+    "href" : "https://fiserviceurlhere/fi/v4/companies/transactiontypes/expense/transactions?limit=3&page=2&size=3",
+    "hreflang" : null,
+    "media" : null,
+    "title" : null,
+    "type" : null,
+    "deprecation" : null
+  } ],
+  "content" : [ {
+    "id" : "e7f810cabc8348cdb051dd9431c8cfbb",
+    "docType" : "expense",
+    "companyId" : "COMPANY_ID_HERE",
+    "entityId" : "ENTITY_ID_HERE",
+    "companyUuid" : "COMPANY_UUID_HERE",
+    "erpSystemId" : "",
+    "document" : "{THE_FINANCIAL_DOCUMENT}",
+    "docStatus" : "READY",
+    "links" : [ ]
+  }, {
+    "id" : "e1500222f16748718fdd2d493ee4c9dd",
+    "docType" : "expense",
+    "companyId" : "COMPANY_ID_HERE",
+    "entityId" : "ENTITY_ID_HERE",
+    "companyUuid" : "COMPANY_UUID_HERE",
+    "erpSystemId" : null,
+    "document" : "{THE_FINANCIAL_DOCUMENT}",
+    "docStatus" : "READY",
+    "links" : [ ]
+  }, {
+    "id" : "e2135678f16748718fdd2d493ee4c9dd",
+    "docType" : "expense",
+    "companyId" : "COMPANY_ID_HERE",
+    "entityId" : "ENTITY_ID_HERE",
+    "companyUuid" : "COMPANY_UUID_HERE",
+    "erpSystemId" : null,
+    "document" : "{THE_FINANCIAL_DOCUMENT}",
+    "docStatus" : "READY",
+    "links" : [ ]
+  }
+   ],
+  "page" : {
+    "size" : 3,
+    "totalElements" : 8,
+    "totalPages" : 3,
+    "number" : 0
+  }
 }
-```
```

Comment:
Put this set of three backticks back in- it makes the example format correctly.

Reply:
Done.